package.json

```diff
@@ -378,6 +381,24 @@
                 "description": "Create a new Draft project",
                 "category": "Draft"
             },
+            {
+                "command": "extension.draftCreateDebug",
+                "title": "Create (debug pack)",
+                "description": "Create a new Draft project using the Node Debug pack",
+                "category": "Draft"
+            },
+            {
+                "command": "extension.draftUpDebug",
+                "title": "up --debug",
+                "description": "Debug the Draft application",
+                "category": "Draft"
+            },
+            {
+                "command": "extension.draftDebug",
+                "title": "Debug",
+                "description": "Debug your Draft application",
```

Comment:
Description "Debug your Draft application" vs `draftUpDebug`'s "Debug the Draft application - hard to know what the difference is.